### DISCUSSION TITLE
Left Stick dinamic bind

### DISCUSSION THREAD

**Discussion by Zugar159** [Key2Joy/Key2Joy#79] [Q&A]
Hello, I would like to know if it's possible to create a command or script that will move the Left Stick to Axys X 14400 while holding down the Z key and return the Left Stick to position X 0 when releasing the Z key. Additionally, can the Z key remain active and send the Z signal to Windows while performing this function?

**Comment by luttje** [comment]
Hey there! Thanks for your interest in this open-source software and question.

First of all: please beware that there's currently some annoying bugs that cause inputs to "stick" or not reach an app. I'm not sure if this is also the case with scripting.

For your question: I think this _may_ be possible with scripting, on which you can read more here: https://github.com/luttje/Key2Joy/blob/main/Docs/Scripting.md

Hope this helps
